rafiki/admin/app.py

```diff
@@ -20,27 +20,62 @@ def index():
 # Users
 ####################################
 
-@app.route('/user', methods=['POST'])
+@app.route('/users/csv', methods=['POST'])
+@auth([UserType.ADMIN])
+def create_users_with_csv(auth):
+    admin = get_admin()
+    params = get_request_params()
+
+    # Expect csv file as bytes
+    csv_file_bytes = request.files['csv_file_bytes'].read()
+    params['csv_file_bytes'] = csv_file_bytes
+
+    with admin:
+        return jsonify(admin.create_users_with_csv(**params))
+
+@app.route('/users', methods=['POST'])
```

Comment:
It is not common to create users using a csv file. Typically, we add users one by one. But we need to get all users for the front-end.
Have you compared users vs user for the operations against a single user? Both are fine, but we need to be consistent with other APIs. E.g., model vs models, trial vs trials.

Reply:
Noted. i will be removing this API